### PR TITLE
Spec file fixes

### PR DESCRIPTION
**Pull request by Conan-Kudo**
This PR contains spec file fixes from myself and @ignatenkobrain from review on `#rpm-ecosystem` IRC discussion.

### CODE REVIEW

File: fedora-core-snap.spec [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-Source0:	.
+Name:           fedora-core-snap
+Version:        0
+Release:        1.git%{shortcommit}%{?dist}
+Summary:        Tools for constructing the fedora-core snap
 
-Requires:	squashfs-tools
-Requires:   dnf
-Requires:   bash
+Group:          Development/Tools
```
Comment (igor-raits) [inline fedora-core-snap.spec]: drop it, bastard!

File: fedora-core-snap.spec [hunk 2]
```diff
@@ -19,18 +21,25 @@ discard unneeded files. The resulting tree is compressed to a squashfs
...
-install -m 0755 mksnap-fedora-core %{buildroot}%{_sbindir}
+install -p -m 0755 mksnap-fedora-core %{buildroot}%{_sbindir}
 
 %files
-%doc LICENSE
+%license LICENSE
+%{_sbindir}/mksnap-fedora-core
 
 %changelog
+* Sun Aug 07 2016 Igor Gnatenko <i.gnatenko.brain@gmail.com> -0-1.0bbaadb
```
Comment (igor-raits) [inline fedora-core-snap.spec]: something is broken in this line.

**Comment by Conan-Kudo** [comment]
Everything should be fixed up now... @zyga? @ignatenkobrain?

File: fedora-core-snap.spec [hunk 3]
```diff
@@ -19,18 +20,25 @@ discard unneeded files. The resulting tree is compressed to a squashfs
...
 %changelog
+* Sun Aug 07 2016 Igor Gnatenko <i.gnatenko.brain@gmail.com> - 0-1.0bbaadb
+- More spec cleanups and fixes
+
+* Sun Aug 07 2016 Neal Gompa <ngompa13@gmail.com> - 1-1.0bbaadb
+- Slight refactor
+
 * Sun Aug 07 2016 Zygmunt Krynicki <me@zygoon.pl> - 1-1
-- Initial version of the package
+- Initial version of the package
```
Comment (igor-raits) [inline fedora-core-snap.spec]: I prefer to have newlines ;)

**Comment by igor-raits** [comment]
lgtm

**Pull request merged**